_inc/lib/admin-pages/class.jetpack-react-page.php

```diff
@@ -136,20 +136,48 @@ function page_render() {
 		/** This action is already documented in views/admin/admin-page.php */
 		do_action( 'jetpack_notices' );
 
-		$static_html = wp_remote_get( esc_url( plugins_url( '/_inc/build/static.html', JETPACK__PLUGIN_FILE ) ) );
-		echo 200 == wp_remote_retrieve_response_code( $static_html )
-			? wp_remote_retrieve_body( $static_html )
-			: esc_html__( 'Error fetching static.html.', 'jetpack' );
+		// Try fetching by patch
+		$static_html = @file_get_contents( JETPACK__PLUGIN_DIR . '_inc/build/static.html' );
+
+		// If it failed, try fetching by URL
+		if ( false === $static_html ) {
+			$static_html = wp_remote_get( esc_url( plugins_url( '/_inc/build/static.html', JETPACK__PLUGIN_FILE ) ) );
```

Comment:
It was using `file_get_contents` initially and I imho don't really think it's necessary. All this started with https://github.com/Automattic/jetpack/issues/4974 and your https://github.com/Automattic/jetpack/pull/4924 so I put this here as a fallback.
Anyway, if you have a suggestion, feel free to push it here.